Makefile

```diff
@@ -144,6 +144,7 @@ test-coverage: ##@tests Run unit and integration tests with covevare
 
 test-e2e: ##@tests Run e2e tests
 	# order: reliability then alphabetical
+	# TODO(tiabc): make a single command out of them adding `-p 1` flag.
```

Comment:
What is `-p 1` flag?

Reply:
It limits the parallel test execution.

Reply:
Yeah. Otherwise tests from different packages will be run in parallel and we'll get weird errors like `datadir is already in use` because we use the same datadir for every test.